recipes-kernel/linux/linux-yocto-onl-linux-6.1.y/arch/arm/boot/dts/arm-accton-as4610-30p.dts

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Commit message "linux-yocto-onl: add linux 6.1 support":
- s/terminating configures VNIs/terminating configured VNIs/
- can you spell out BSP?

Reply:
> can you spell out BSP?

I can't from the top of my head (I could guess), but I can extend it to KCONF_BSP_AUDIT, is that enough?

Reply:
I was hoping for something that has less than 198.000.000 Google hits, but more than zero.

Reply:
I do. I would just like the commit message to be useful to people who don't.